battle-service/src/main/java/by/ibondarau/tictactoe/battleservice/controller/BattleController.java

```diff
@@ -13,88 +20,97 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.time.ZonedDateTime;
-import java.util.ArrayList;
-import java.util.Collections;
+import java.util.EnumSet;
 import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
 
 @RestController
 @RequestMapping("/battle")
+@RequiredArgsConstructor
+@Slf4j
 public class BattleController {
 
+    private final BattleService battleService;
+    private final ModelMapper modelMapper;
+
     @GetMapping("/{battleId}")
-    public ResponseEntity<BattleResponseDto> get(@PathVariable Integer battleId) {
-        BattleResponseDto dto = new BattleResponseDto()
-                .id(battleId)
-                .firstPlayerId(1)
-                .secondPlayerId(2)
-                .size(3)
-                .status("Finished")
-                .moves(Collections.emptyList())
-                .nextMove(null)
-                .result("First wins")
-                .created(ZonedDateTime.now().toInstant())
-                .started(ZonedDateTime.now().toInstant())
-                .finished(ZonedDateTime.now().toInstant());
-        return ResponseEntity.ok(dto);
+    public ResponseEntity<BattleResponseDto> get(@PathVariable UUID battleId) {
+        return ResponseEntity.ok(modelMapper.battleToBattleResponse(battleService.getBattle(battleId)));
     }
 
     @PostMapping
     public ResponseEntity<BattleResponseDto> createBattle(@RequestBody CreateBattleDto createBattleDto) {
-        BattleResponseDto dto = new BattleResponseDto()
-                .id(1)
-                .firstPlayerId(createBattleDto.userId())
-                .secondPlayerId(2)
-                .size(3)
-                .status("Created")
-                .moves(Collections.emptyList())
-                .nextMove(1)
-                .result(null)
-                .created(ZonedDateTime.now().toInstant())
-                .started(null)
-                .finished(null);
-        return ResponseEntity.ok(dto);
+        log.debug("""
```

Comment:
we talked about adding this logging in separate filter, as i remember

Reply:
Implemented, see LoggingFilter

Reply:
After re-testing of the application I found out that I cannot read request body in filters because I read body with reader reader and it's already been read by the Spring itself before coming to the filter. Besides, I found a lot of useful spring by-default logs, including request logging, so I decided to abandon the idea of custom request logging filter and remove it from the application.